ComponentKit/StatefulViews/CKStatefulViewReusePool.mm

```diff
@@ -56,6 +90,8 @@ void addView(UIView *view)
 @implementation CKStatefulViewReusePool
 {
   std::unordered_map<std::pair<__unsafe_unretained Class, id>, FBStatefulReusePoolItem, PoolKeyHasher> _pool;
+  std::unordered_map<std::pair<__unsafe_unretained Class, id>, FBStatefulReusePoolItem, PoolKeyHasher> _pendingPool;
```

Comment:
Maybe name this to better indicate what's pending. `_pendingReusePool` is probably better. I may even rename `_pool` to `_reusePool` as well. Up to you.